src/test/java/seatsio/charts/ListChartsTest.java

```diff
@@ -100,4 +102,20 @@ public void pageSize() {
                 .containsExactly(chart3.key, chart2.key);
     }
 
+    @Test
+    public void listChartsWithValidationTest() {
+        createTestChartWithErrors();
```

Comment:
I would only create one chart, and assert that the validation errors are equal to the expected validation errors

Reply:
Can you take another look please?